README.md

```diff
@@ -40,10 +44,10 @@ docker pull benjixd/dietstory-build
 
 ### Building the Server
 
-The following command compiles the Dietstory project, replacing $JAVA_HOME with your Java 7 path:
+The following command compiles the Dietstory project, replacing $JAVA_HOME with your Java 8 path:
 
 ```
-ant -Dplatforms.JDK_1.7.home=$JAVA_HOME -Dnb.internal.action.name=rebuild clean jar
+ant -Dplatforms.JDK.home=$JAVA_HOME -Dnb.internal.action.name=rebuild clean jar
```

Comment:
```suggestion
ant -Dplatforms.JDK.home="$JAVA_HOME" -Dnb.internal.action.name=rebuild clean jar
```
This supports Windows users better because of `Program Files`

Reply:
It has been updated